### PR TITLE
tidy up shared info for bastion

### PR DESCRIPTION
**Pull request by jianlinliu**
1. remove duplicated proxy-config-generate ref
2. use consistent file name for bastion host to save info

### CODE REVIEW

File: ci-operator/step-registry/aws/provision/bastionhost/aws-provision-bastionhost-commands.sh [hunk 1]
```diff
@@ -429,10 +429,17 @@ echo "${INSTANCE_ID}" >> "${SHARED_DIR}/aws-instance-ids.txt"
...
 BASTION_HOST_PUBLIC_DNS="$(aws --region "${REGION}" cloudformation describe-stacks --stack-name "${stack_name}" \
   --query 'Stacks[].Outputs[?OutputKey == `PublicDnsName`].OutputValue' --output text)"
+BASTION_HOST_PRIVATE_DNS="$(aws --region "${REGION}" cloudformation describe-stacks --stack-name "${stack_name}" \
+  --query 'Stacks[].Outputs[?OutputKey == `PrivateDnsName`].OutputValue' --output text)"
 
-echo "${BASTION_HOST_PUBLIC_DNS}" > "${SHARED_DIR}/bastion_dns"
-PROXY_URL="http://${PROXY_CREDENTIAL}@${BASTION_HOST_PUBLIC_DNS}:3128"
-echo "${PROXY_URL}" > "${SHARED_DIR}/public_proxy_url"
+echo "${BASTION_HOST_PUBLIC_DNS}" > "${SHARED_DIR}/bastion_public_address"
+echo "${BASTION_HOST_PRIVATE_DNS}" > "${SHARED_DIR}/bastion_private_address"
```
Comment (jianli-wei) [inline ci-operator/step-registry/aws/provision/bastionhost/aws-provision-bastionhost-commands.sh:436]: Sorry, the saved data would be DNS or IP? If DNS, why both public DNS and private DNS?

**Review by jianli-wei — CHANGES_REQUESTED** [review]

File: ci-operator/step-registry/aws/provision/bastionhost/aws-provision-bastionhost-commands.sh (hunk 1 above)
Comment (jianlinliu) [reply ci-operator/step-registry/aws/provision/bastionhost/aws-provision-bastionhost-commands.sh:436]: > the saved data would be DNS or IP

This depends on the ref author, any one is okay, in aws case, it is DNS.

> both public DNS and private DNS
Different cases use different items, inside cluster, the private one should be used, outside cluster, the public one should be used.

**Comment by Amoghrd** [comment]
Seems like as you mentioned `-` in a file name does not have issues

**Comment by jianlinliu** [comment]
> Seems like as you mentioned `-` in a file name does not have issues

Yeah, two same ref yaml files are created there, so remove one of them.

**Pull request merged**